api.planx.uk/modules/auth/strategy/microsoft-oidc.ts

```diff
@@ -41,18 +42,30 @@ export const getMicrosoftOidcStrategy = (client: Client): Strategy<Client> => {
 };
 
 const verifyCallback: StrategyVerifyCallbackReq<Express.User> = async (
-  req: Http.IncomingMessageWithSession,
+  req: Http.IncomingMessageWithCookies,
   tokenSet,
   done,
 ): Promise<void> => {
   // TODO: use tokenSet.state to pass the redirectTo query param through the auth flow
+  // TODO: validate id_token sig with the public key from the jwks_uri (...v2.0/keys)
```

Comment:
It's not especially clear how to do this using `openid-client` v5x, but they completely rewrote the package for v6 and that might offer a clearer route to fetching the keys and verifying the signature

Related task in icebox: https://trello.com/c/7BMpYwyt